Gosu/Image.h

```diff
@@ -25,8 +25,8 @@ Gosu_Image* Gosu_Image_create_from_subimage(Gosu_Image *source, int left, int to
 void Gosu_Image_destroy(Gosu_Image *image);
 
 // Image properties
-int Gosu_Image_width(Gosu_Image *image);
-int Gosu_Image_height(Gosu_Image *image);
+const unsigned Gosu_Image_width(Gosu_Image *image);
```

Comment:
The const here does not make a difference since you cannot assign to a return value -> unsigned or unsigned int is enough.